resources/public/api/conf/1.0/lossClaims_amend.raml

```diff
@@ -0,0 +1,25 @@
+is:
+  - acceptJson
+  - contentTypeJson
+  - testScenarioHeader
+  - formatNino
+  - formatClaimId
+  - incorrectOrEmptyBody
+  - formatTypeOfClaim
+  - ruleTypeOfClaim
```

Comment:
RULE_CLAIM_TYPE_NOT_VALID vs RULE_TYPE_OF_CLAIM_INVALID

Reply:
Would RULE_CLAIM_TYPE_INVALID be better maybe? (combine the two)